playbooks/roles/local.defaults/templates/config.yml.j2

```diff
@@ -97,5 +97,5 @@ config:
     {%- endfor +%}
   {%- endfor +%}
 
-  statedir: "{{ misc.host.statedir }}/sit.{{ backend }}_statedump"
+  statedir: "{{ misc.host.statedir }}/sit.{{ be if not variant else be + '.' + variant }}_statedump"
```

Comment:
The benefit on top of my head would be to differentiate between various tarballs(based on backend and/or variant) downloaded at the same location as part of some analysis where it will be difficult to say which is which.

Reply:
The artifact is downloaded from a link inside the job, which is already associated with a particular backend and variant. This is enough information for downloading. On the other side, if we want to fully identify the tests, we would need the date and/or commit version used for samba, ceph, gluster, ... (and maybe other details). It doesn't make sense to keep all this information in the file/directory name.

I think it would be more useful to have the same directory and file names, and add an additional file which contains more detailed information about the backend and variant, when it was executed, the versions of each component, ...

In my experience, having unpredictable names only makes automated scripts more complex and error-prone. I prefer to have the same known structure everywhere and use different subdirectories or other methods to separate the data when necessary.

Reply:
> The artifact is downloaded from a link inside the job, which is already associated with a particular backend and variant. This is enough information for downloading.

I was talking more about looking at statedump contents for various backends(and probably varaints) already downloaded at same location(locally). As of now the only differentiation is in the top directory name.

>  On the other side, if we want to fully identify the tests, we would need the date and/or commit version used for samba, ceph, gluster, ... (and maybe other details). It doesn't make sense to keep all this information in the file/directory name. I think it would be more useful to have the same directory and file names, and add an additional file which contains more detailed information about the backend and variant, when it was executed, the versions of each component, ...

I agree but this change has nothing to do with those details.

> In my experience, having unpredictable names only makes automated scripts more complex and error-prone. I prefer to have the same known structure everywhere and use different subdirectories or other methods to separate the data when necessary.

I don't think the current directory name has any unpredictable factor in it. But if you insist I can change it to _sit_statedump_.